docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
+
+* The job is triggered by the Fedora Messaging messages which:
+
+  * Indicate that a pull request was opened, re-opened, or synchronized (“synchronized” meaning that code changes were pushed to an already-opened pull request)
+
+    Ex: [PR-opened](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.pull_request.opened)
+
+  * Indicate a comment was added that contains the text `/retest` or `/test CVP`
+
+    Ex: [PR-comment-added](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.issue.comment&delta=3600)
+
+* The openshift project for the `test-operator-test-playbooks` is `cvp-ci-operatortestplaybooks`
+
+* The test-operator-test-playbooks job parses the fedmsg to determine what files have changed and what repository and branch need to be tested.
+
+* If the data can be successfully parsed, test-operator-test-playbooks uses `CVP bot account` on GitHub to create a check status, `CVP/pr-sanity-test`, for the pull request.
+
+* Note: If the only file that has changed is README.md, test-operator-test-playbooks will then exit successfully. No further tests are required since no ansible playbooks have changed.
+
+* The test-operator-test-playbooks job next configures it’s own [Team Jenkins server](https://jenkins-cvp-ci-operatortestplaybooks.cloud.paas.psi.redhat.com/) to run the testing pipelines.
+
+* When this is complete the test-operator-test-playbooks currently triggers three test pipelines on it’s team Jenkins server:
+
+  * cvp-redhat-operator-bundle-image-validation-test
+  * cvp-isv-operator-metadata-validation-test
+  * cvp-redhat-operator-metadata-validation-test
+
+* These are the same pipelines that run in production. The operators used for testing in these pipelines are the same used in [CVP-CI setup tests](https://gitlab.sat.engineering.redhat.com/cvp/pipeline/-/tree/master/ci/tests/setupTests/sample-ci-messages).
```

Comment:
There should not be any internal links in upstream documentation.
It is unfair for the community.